packages/app/src/providers/mock/mock-api.ts

```diff
@@ -10,25 +9,27 @@ import type {
   AttributeValues,
   Dataset,
   Entity,
+  GroupWithChildren,
   NumArrayDataset,
   ProvidedEntity,
   Value,
 } from '@h5web/shared/hdf5-models';
-import type { MockFile } from '@h5web/shared/mock/models';
 import {
   assertMockAttribute,
   assertMockDataset,
-} from '@h5web/shared/mock/utils';
+} from '@h5web/shared/mock-utils';
 import axios from 'axios';
 
 import { DataProviderApi } from '../api';
 import type { ExportFormat, ExportURL, ValuesStoreParams } from '../models';
 import { sliceValue } from '../utils';
+import { makeMockFile } from './mock-file';
+import { findMockEntity } from './utils';
 
 export const SLOW_TIMEOUT = 3000;
 
 export class MockApi extends DataProviderApi {
-  private readonly mockFile: MockFile;
+  private readonly mockFile: GroupWithChildren;
```

Comment:
I removed this alias after all, since it's only used here and in `makeMockFile`, and since the latter is now in this folder as well, I didn't fee like creating a `models.ts` file just for that... :sweat_smile: